logpyle/runalyzer.py

```diff
@@ -44,8 +44,6 @@ def get_rank_agg_table(self, qty, rank_aggregator):
         if (qty, rank_aggregator) in self.rank_agg_tables:
             return tbl_name
 
-        print("INFO: building temporary rank aggregation table %s" % tbl_name)
```

Comment:
How about replacing this with `logger.info` instead?

Reply:
Is this message important enough to log it? I can only imagine that building this table might cause a (small?) delay.

Reply:
My sense was that this could take a long time, otherwise there wouldn't be a message. Certainly long enough for folks to get confused. Maybe a use case for `pytools.ProcessLogger`?

Reply:
I've changed it to `logger.info`. Since this is supposed to make the output more predictable for automated use of `runalyzer`, `pytools.ProcessLogger` isn't appropriate here imho.